events/moose/moose.go

```diff
@@ -234,21 +234,16 @@ func (s *Subscriber) NotifyIpv6(data bool) error {
 
 func (s *Subscriber) NotifyLogin(any) error { return nil }
 
-func (s *Subscriber) NotifyRate(data events.ServerRating) error {
-	return s.response(moose.NordvpnappSendUserInterfaceUiItemsClick(
-		"server_speed_rating",
-		moose.NordvpnappUserInterfaceItemTypeButton,
-		data.Server,
-		fmt.Sprintf("%d", data.Rate),
-	))
-}
-
-func (s *Subscriber) NotifySendInvitation(any) error {
+func (s *Subscriber) NotifyUiItemsClick(data events.UiItemsAction) error {
+	itemType := moose.NordvpnappUserInterfaceItemTypeButton
+	if data.ItemType == "textbox" {
```

Comment:
wouldn't be possible to have `"textbox"` defined into a constant or an enum, since it is used also in the server.go file?

Reply:
These are defined by enumerated constants, but in file which is conditionally imported only for moose-enabled builds. So I think it is better to just use strings, instead of copying that enum to our code and later to having to keep it manually in sync with the moose library.